docs/settings.rst

```diff
@@ -332,3 +336,59 @@ A boolean indicating whether or not to require clients to be authenticated. May
 .. code-block:: python
 
   "REQUIRE_CLIENT_AUTH": True
+
+
+.. _settings-filter-behavior:
+
+FILTER_BEHAVIOR
+^^^^^^^^^^^^^^^^^^^
+
+Variable allowing user to configure how the filter work.
+
+Values possible are: 
+
+- ``METADATA_STRICT`` that only allow filtering against metadata. This is the default behavior as it's the legacy way of filtering. This will change for 1.0.0. This mode doesn't add additional field into the messages
+- ``REQUEST_STRUCT_STRICT`` that only allow filtering against request ``_filters`` field. This mode add the ``_filters`` key in every message except if service specifically disable it.
+- ``METADATA_AND_REQUEST_STRUCT`` that allow filtering against metadata and request ``_filters`` field. This mode add the ``_filters`` key in every message except if service specifically disable it. If filter is present in both metadata and _filters field, the one in _filters fiels have priority
+
+
+The mains differences between metadata and request are:
+- Metadata are not specified in the proto file so you add or removing filtering possibility without deploying a new api verison or causing breaking change
+- Request allow you to help the developper understand which endpoint accept filtering and automatically document it
+- Request are serialized so it can improve performance if filtering with large amount of data
+
+An util class exist to help you set this settings:
+
+.. code-block:: python
```

Comment:
```suggestion
The mains differences between metadata and request are:
- Metadata are not specified in the proto file so adding or removing a filter in a new version is not documented and may cause breaking changes.
- Requests allows helping developpers understand which endpoint accept filtering and automatically document it
- Requests are serialized so it can improve performance when filtering large amount of data

.. code-block:: python
```